spec/cc/engine/analyzers/javascript/main_spec.rb

```diff
@@ -91,8 +95,8 @@
           <a className='button button-primary full' href='#' onClick={this.onSubmit.bind(this)}>Login</a>
     EOJSX
 
-    result = run_engine(engine_conf).strip
-    issues = result.split("\0")
+    issues = run_engine(engine_conf).strip.split("\0")
```

Comment:
I'd back this out. It's not a *terrible* case of drive-by, but just to flex the "essential changes only" muscles.